lib/nvml.ml

```diff
@@ -84,8 +85,14 @@ external device_get_active_vgpus: interface -> device -> vgpu_instance list =
 external vgpu_instance_get_vm_domid: interface -> vgpu_instance -> vm_domid = 
 	"stub_nvml_vgpu_instance_get_vm_id"
 
+external get_vgpu_metadata: interface -> vgpu_instance -> vgpu_metadata =
+	"stub_nvml_get_vgpu_metadata"
+
 external get_pgpu_vgpu_compatibility: interface -> vgpu_instance -> pgpu_metadata -> vgpu_compatibility_t =
-    "stub_nvml_get_pgpu_vgpu_compatibility" 
+	"stub_nvml_get_pgpu_vgpu_compatibility"
+external get_pgpu_vgpu_compatibility2: interface -> vgpu_metadata -> pgpu_metadata -> vgpu_compatibility_t =
```

Comment:
Maybe we should just have this and replace the other one with a call to `get_vgpu_metadata` and `get_pgpu_vgpu_compatibility2`. Even because I think we only use it indirectly in the call to  `get_pgpu_vm_compatibility` in xapi.

It may also be worth renaming both functions to `get_pgpu_vgpu_instance_compatibility` and `get_pgpu_vgpu_metadata_compatibility` or something else that clarifies more the difference between them.